### PR TITLE
fixed procedure to generate EAST model in IR format

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: docs/east_ocr.md [hunk 1]
```diff
@@ -29,26 +29,62 @@ The metadata are the `text_coordinates` and the `confidence_level` outputs.
...
+git clone https://github.com/argman/EAST 
+cd EAST 
+```
+Download and unzip the fle east_icdar2015_resnet_v1_50_rbox.zip to EAST folder with the github repository.
 ```bash
 unzip ./east_icdar2015_resnet_v1_50_rbox.zip
 ```
-Convert the TensorFlow Model to Intermediate Representation format using the model_optimizer tool. It can be done
-using a public docker image with OpenVINO toolkit:
+Inside the EAST folder add a file `freeze_east_model.py` with the following content:
```
Comment (dkalinowski) [inline docs/east_ocr.md:42]: Maybe instead adding this to .md file add it to separate .py file and refer to this script?

**Review by dkalinowski — APPROVED** [review]

File: src/custom_nodes/east_ocr/README.md [hunk 2]
```diff
@@ -45,3 +45,5 @@ It will compile the library inside a docker container and save the results in `l
 | overlap_threshold | a ratio in a range of 0-1 for non-max suppression algorithm. Defines the overlapping ratio to reject detection as duplicated  | 0.3 |
 | debug  | Defines if debug messages should be displayed | false |
 | max_output_batch  | Prevents too big batches with incorrect confidence level. It can avoid exceeding RAM resources | 100 |
+| x_corner_adjustment | Horizontal size expansion level for text images to compensate cut letter | 0 | 
+| y_corner_adjustment | Vertical size expansion level for text images to compensate cut letter | 0 |
```
Comment (mzegla) [inline src/custom_nodes/east_ocr/README.md:49]: I'm not sure about that `to compensate cut letter` part. I understand that the model cuts part of the picture and therefore does not recognize the whole text right? I would rephrase that to perhaps give a little bit more context. 

Also what are the units of the adjustment? Pixels? Percentage?

Comment (dkalinowski) [reply src/custom_nodes/east_ocr/README.md:49]: For x percentage of box width, for y percentage of box height.

Comment (dkalinowski) [reply src/custom_nodes/east_ocr/README.md:49]: Also there is mistake here, default values are:
- x_corner_adjustment: 0.12
- y_corner_adjustment: 0.3

Comment (dkalinowski) [reply src/custom_nodes/east_ocr/README.md:49]: Should we change it in source to 0.0 and set defaults for demo in example config.json? @dtrawins

**Review by mzegla — APPROVED** [review]
